### ISSUE TITLE
Anasazi: LOBPCGEpetraExSimple.cpp with CMakeLists.txt

### ISSUE THREAD

**Issue by glanzkaiser** [trilinos/Trilinos#10599]
HI all,

I was reading this:
[anasazi](https://trilinos.github.io/pdfs/anasazi-ug-public.pdf)

at page 10 there is a Makefile, why it can't work? I tried to create the Makefile, but it is not working. Is it because it has been published long time ago?

Then I try to use this .cpp file:
[LOBPCG](https://github.com/trilinos/Trilinos/blob/64d50199f342860b00d16cb78cb5730de66438b9/packages/anasazi/epetra/example/LOBPCG/LOBPCGEpetraExSimple.cpp)

I create CMakeLists.txt here:
[CMake1](https://github.com/glanzkaiser/glanzshamzs/blob/main/Trilinos/Anasazi/example/CMakeLists.txt)

For building the LOBPCG file. But an error occur. What is wrong with my CMakeLists?

I see the CMakeLists from the source directory of Anasazi:
[CMakesource](https://github.com/trilinos/Trilinos/blob/64d50199f342860b00d16cb78cb5730de66438b9/packages/anasazi/epetra/example/LOBPCG/CMakeLists.txt)

I try CMakeLists from the source directory, but it prompts an error:
"Unknown CMake command: TRIBITS_ADD_EXECUTABLE"

Thank You.

![Bildschirmfoto_2022-06-04_19-47-08](https://user-images.githubusercontent.com/72222484/172000187-b2a9c5b9-8a00-4eb3-a804-795a1040d934.png)
![Bildschirmfoto_2022-06-04_19-47-18](https://user-images.githubusercontent.com/72222484/172000189-5a69faf9-8a2e-46db-b5bd-4ded915abf56.png)

**Comment by jjellio** [comment]
Can you post the output of `make VERBOSE=1`

You can't use the CMakeLists.txt from Trilinos' source directly. It's a TRIBITS project (an abstraction ontop of CMake).

At first glance, your CMakeList.txt that *you* wrote, looks mostly okay.  I suspect you need to set your compiler flags to match Trilinos'

```

# this needs to happen before project is called
SET(CMAKE_CXX_FLAGS  "${Trilinos_CXX_COMPILER_FLAGS} ${CMAKE_CXX_FLAGS}" )
SET(CMAKE_C_FLAGS  "${Trilinos_C_COMPILER_FLAGS} ${CMAKE_C_FLAGS}"  )
SET(CMAKE_Fortran_FLAGS  "${Trilinos_Fortran_COMPILER_FLAGS} ${CMAKE_Fortran_FLAGS}"  )

project(YourProject)

```

**Comment by glanzkaiser** [comment]
I do not comprehend TRIBITS at all. 
Why TRIBITS is used for CMake for Trilinos?

I have made it, thanks @jjellio ... 
**Solution:** I added Anasazi at my CMakeLists. Which one is better to compile for Trilinos solver / package using Cmakelists.txt or Makefile directly?

Thank You

![Bildschirmfoto_2022-06-05_13-56-17](https://user-images.githubusercontent.com/72222484/172039340-830a39c0-9893-45b5-a042-85fdb7d4a236.png)
